### ISSUE TITLE
Add possibility to select the fixers we want to run

### ISSUE THREAD

**Issue by stof** [umpirsky/Symfony-Upgrade-Fixer#31]
This tool does not allow disabling some fixers. This makes it harder to use. For instance, FormGetnameToGetblockprefixFixer must run only after you replace all usages of your type names by class names when adding fields. As you tool is only able to upgrade core form types, it means it leaves the project in a broken state by default.

**Comment by umpirsky** [comment]
Yes, I know. The idea was to make this work before `v1.0`, unfortunately I did not have enough time to dedicate to the project. 

For the time being, I'm just trying to review and merge PRs.

Thanks!

**Comment by stof** [comment]
Well, the infrastructure is based on php-cs-fixer. And this tool has the ``--fixers`` option in its command-line. The work would be to port this feature too

**Comment by umpirsky** [comment]
Yes.
